opentelemetry-otlp/src/exporter/mod.rs

```diff
@@ -217,6 +268,34 @@ impl<B: HasExportConfig> WithExportConfig for B {
         self.export_config().endpoint = exporter_config.endpoint;
         self.export_config().protocol = exporter_config.protocol;
         self.export_config().timeout = exporter_config.timeout;
+        #[cfg(feature = "tls")]
+        {
+            self.export_config().insecure = Some(true);
```

Comment:
As per the specs, the connection should be secure by default.
https://github.com/open-telemetry/opentelemetry-specification/blob/3b037be082915f29766a87c01141c72fb0fb5f1b/specification/protocol/exporter.md

![image](https://github.com/user-attachments/assets/0675adf8-0f22-4281-b163-63f62c4de587)

```